save-demo/src/main/kotlin/com/saveourtool/save/demo/storage/DependencyStorage.kt

```diff
@@ -0,0 +1,168 @@
+package com.saveourtool.save.demo.storage
+
+import com.saveourtool.save.demo.config.ConfigProperties
+import com.saveourtool.save.demo.entity.Demo
+import com.saveourtool.save.demo.entity.Dependency
+import com.saveourtool.save.demo.repository.DependencyRepository
+import com.saveourtool.save.s3.S3Operations
+import com.saveourtool.save.storage.AbstractStorageWithDatabaseEntityKey
+import com.saveourtool.save.storage.concatS3Key
+import com.saveourtool.save.utils.*
+import org.slf4j.Logger
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import java.nio.ByteBuffer
+import java.nio.file.Path
+import kotlin.io.path.*
+
+private const val TOOLS_PATH = "tools"
```

Comment:
Probably should be changed to `dependencies` or `deps` for short

Reply:
changed to `deps`